accurest-core/src/main/groovy/io/coderate/accurest/dsl/WiremockStubStrategy.groovy

```diff
@@ -0,0 +1,86 @@
+package io.coderate.accurest.dsl
+
+import groovy.json.JsonOutput
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class WiremockStubStrategy {
+
+    private final Request request
+
+    WiremockStubStrategy(GroovyDsl groovyDsl) {
+        this.request = groovyDsl.request
+    }
+
+    String toWiremockClientStub() {
```

Comment:
`toClient*` and `toServer*`are the another dimension. Maybe they could be also extracted and call from WiremockStubStrategy?